metadata/config.yaml

```diff
@@ -50,6 +50,9 @@ CAPELLA_FORK_EPOCH: 194048  # April 12, 2023, 10:27:35pm UTC
 # Deneb
 DENEB_FORK_VERSION: 0x04000000
 DENEB_FORK_EPOCH: 269568  # March 13, 2024, 01:55:35pm UTC
+# Electra
+ELECTRA_FORK_VERSION: 0x05000000
+ELECTRA_FORK_EPOCH: 364032  # May 7, 2025, 10:05:11am UTC
```

Comment:
Don't we also "need" the new Electra config values like:

https://github.com/ethereum/consensus-specs/blob/05850cc4fe93ae1f1a182ebf7629eacdacb731a9/configs/mainnet.yaml#L156-L166

For Lighthouse it doesn't matter, we have defaults, but I think it pays to be explicit, and some clients may require this.

Reply:
Oh yeah. You're right. Good catch.

Reply:
I added these + a few more necessary changes here. A PR into this branch.

* https://github.com/eth-clients/mainnet/pull/5